tests/functional/a/abstract/abstract_abc_methods.py

```diff
@@ -1,9 +1,9 @@
 """ This should not warn about `prop` being abstract in Child """
-# pylint: disable=too-few-public-methods, useless-object-inheritance
+# pylint: disable=too-few-public-methods
 
 import abc
 
-class Parent(object):

```

Comment:
We need to keep **some** tests for ``useless-object-inheritance`` this is a python 3 checks : habits die hard and what was necessary in python 2 is not anymore but people keep doing it.

Reply:
Does [useless-object-inheritance.py ](https://github.com/PyCQA/pylint/blob/main/tests/functional/u/useless/useless_object_inheritance.py)cover it already?

Reply:
Sure ! I did not check, sorry 😄